docs/core/tools/dotnet-new.md

```diff
@@ -126,6 +163,61 @@ The command contains a default list of templates. Use `dotnet new -all` to obtai
 
 ## Options
 
+# [.NET Core 2.2](#tab/netcore22)
+
+`--dry-run`
+
+Displays a summary of what would happen if the given command were run if it would result in a template creation.
+
+`--force`
+
+Forces content to be generated even if it would change existing files. This is required when the output directory already contains a project.
+
+`-h|--help`
+
+Prints out help for the command. It can be invoked for the `dotnet new` command itself or for any template, such as `dotnet new mvc --help`.
+
+`-i|--install <PATH|NUGET_ID>`
+
+Installs a source or template pack from the `PATH` or `NUGET_ID` provided. If you want to install a prerelease version of a template package, you need to specify the version in the format of `<package-name>::<package-version>`. By default, `dotnet new` passes \* for the version, which represents the last stable package version. See an example at the [Examples](#examples) section.
+
+For information on creating custom templates, see [Custom templates for dotnet new](custom-templates.md).
+
+`-l|--list`
+
+Lists templates containing the specified name. If invoked for the `dotnet new` command, it lists the possible templates available for the given directory. For example if the directory already contains a project, it doesn't list all project templates.
+
+`-lang|--language {C#|F#|VB}`
+
+The language of the template to create. The language accepted varies by the template (see defaults in the [arguments](#arguments) section). Not valid for some templates.
+
+> [!NOTE]
+> Some shells interpret `#` as a special character. In those cases, you need to enclose the language parameter value, such as `dotnet new console -lang "F#"`.
+
+`-n|--name <OUTPUT_NAME>`
+
+The name for the created output. If no name is specified, the name of the current directory is used.
+
+`--nuget-source`
+
+Specifies a NuGet source to use during install.
+
+`-o|--output <OUTPUT_DIRECTORY>`
+
+Location to place the generated output. The default is the current directory.
+
+`--type`
+
+Filters templates based on available types. Predefined values are "project", "item" or "other".
+
+`-u|--uninstall <PATH|NUGET_ID>`
+
```

Comment:
Perhaps add a note about what happens when running `dotnet new -u` without a value?

The behavior is to show currently installed packs, with the templates listed under each pack. I find it useful for when I want to uninstall something,  it shows exact uninstall string.